webapp/templates/jury/tsv/accounts.tsv.twig

```diff
@@ -0,0 +1,4 @@
+accounts	1
+{% for row in data %}
+{{ row.type|replace({'\t':' '}) }}	{{ row.fullname|replace({'\t':' '}) }}	{{ row.username|replace({'\t':' '}) }}	{{ row.password|replace({'\t':' '}) }}
```

Comment:
This is modifying data in an unclear way: if indeed a field contained a tab, then this may lead to weird and difficult to trace problems. It seems better to me to replace that tab by an escaped tab like a literal `\t`. That way, it's at least clearer what's going on.

Reply:
(Note that this just mirrors what the previous export did, but the point is valid.)

I propose that we fix it by following this standard "convention" which is wat Jaap says extended to other problematic symbols. Put that into a twig filter and apply it to our tsv exports.
https://en.wikipedia.org/wiki/Tab-separated_values#Conventions_for_lossless_conversion_to_TSV

And I think it'd be great if the CLICS spec would mention explicitly how this is expected to be handled for all tsv's (and obviously preferedly with the convention as specified above)